apps/wpcom-block-editor/src/wpcom/features/domain-upsell-callout.scss

```diff
@@ -0,0 +1,69 @@
+@import "@automattic/typography/styles/variables";
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+
+.domain-upsell-callout {
```

Comment:
I'd probably prefix this with `jetpack_` or something to avoid clashing with anything. I know it's unlikely but WP-admin can have literally anything from any plugin.